fix-android-dependencies.py

```diff
@@ -0,0 +1,109 @@
+import json
```

Comment:
please name this `fix_android_dependencies.py` so that it's a proper python module name. (it'd be great to have tests as well, but I'm not overly concerned about it)